backend/node/genvm/base.py

```diff
@@ -133,9 +134,59 @@ async def run_contract(
     def get_contract_schema(contract_code: str) -> dict:
 
         namespace = {}
-        exec(contract_code, globals(), namespace)
+        try:
+            exec(contract_code, globals(), namespace)
+        except SyntaxError as err:
```

Comment:
Exception handling looks great :100: 

Could you provide some docs as comments? I mean around how to get these types of errors and the `sys` module
The part with the extract of information is very cryptic at first glance.